defaults/main.yml

```diff
@@ -1,3 +1,12 @@
 ---
-dhcp_group: "dhcp_only_hosts"
-dhcp_pxe_group: "dhcp_pxe_hosts"
+lang: "en_US.UTF-8"
```

Comment:
This is a matter of preference, but generally I prefer to prefix all role-specific variables with a role-specific prefix. I have several times run into issues where variable names are accidentally overloaded, and this generates hard to debug problems. Having a prefix makes it better, even though it doesn't remove the problem.

So, e.g. all variables could be kickstart_. This of course touches a lot of files.